lib/transfers.js

```diff
@@ -472,12 +472,15 @@ export class Transfers {
    * @example
    * const moov = new Moov(...);
    * try {
-   *   const { transferID } = moov.transfers.refund("...");
+   *   const { transferID } = moov.transfers.refund(
+   *    "...",
+   *    { amount: 200 } // omit if creating a refund for the full amount
+   *  );
    * } catch (err) {
    *   // ...
    * }
    */
-  async refund(transferID, idempotencyKey) {
+  async refund(transferID, idempotencyKey, refund) {
```

Comment:
```async refund(transferID, idempotencyKey, refund) {```
Should be `refund?` as it's an optional parameter! 
It's also convenient to be able to POST no json to just refund to entire transfer. 
Thanks !